.github/workflows/testing.yml

```diff
@@ -0,0 +1,42 @@
+name: Visual regression testing
```

Comment:
For now this only runs if you add the run_vrt label and it has the `skip` flag set to true so it will just return passed automatically without having to run it yet.